.github/actions/setup-node-npm-ci/action.yaml

```diff
@@ -0,0 +1,22 @@
+name: Setup Node 20 and run `npm ci`
+description: Uses specified Node version and runs npm commands to set up the environment for REST API CI
```

Comment:
```suggestion
```

I'd remove this unless it's required, I think it's redundant with the `name`

Reply:
It's required by schema but it's not actually required by the system. We should keep it because schema enforcement could become more strict over time.